src/discourseGraphsMode.ts

```diff
@@ -748,7 +750,36 @@ const initializeDiscourseGraphsMode = async (args: OnloadArgs) => {
 
       window.roamAlphaAPI.ui.commandPalette.addCommand({
         label: "Export Discourse Graph",
-        callback: () => exportRender({}),
+        callback: () => {
+          const discourseNodes = getDiscourseNodes().filter(
+            (r) => r.backedBy !== "default"
+          );
+          const results = (isSamePageEnabled: boolean) =>
+            Promise.all(
+              discourseNodes.map((d) =>
+                fireQuery({
+                  returnNode: "node",
+                  conditions: [
+                    {
+                      relation: "is a",
+                      source: "node",
+                      target: d.type,
+                      uid: window.roamAlphaAPI.util.generateUID(),
+                      type: "clause",
+                    },
+                  ],
+                  selections: [],
+                  isSamePageEnabled,
+                })
+              )
+            ).then((r) => r.flat());
+          exportRender({
+            results,
+            exportTypes: getExportTypes({
+              results,
+            }),
+          });
```

Comment:
API could probably be simplified further

Reply:
Also, from this point forward we should look to move logic out of `discourseGraphsMode` to back within the QueryBuilder extension at large. We then would want to nuke this option, and the Discourse Graph grammar is a downloadable schema from the grammar registry we talked about